public/app/features/dashboard/state/getPanelOptionsWithDefaults.ts

```diff
@@ -160,7 +160,7 @@ function adaptFieldColorMode(
 
     // When supporting value colors and prefering thresholds, use Thresholds mode.
     // Otherwise keep current mode
-    if (colorSettings.byValueSupport && colorSettings.preferThresholdsMode) {
+    if (colorSettings.byValueSupport && colorSettings.preferThresholdsMode && mode?.id !== FieldColorModeId.Fixed) {
```

Comment:
@torkelo -- this will always keep fixed when switching panels, is that reasonable?